flow/core/experiment.py

```diff
@@ -148,8 +149,12 @@ def rl_actions(*_):
         # time profiling information
         t = time.time()
         times = []
+
+        # data pipeline
         extra_info = defaultdict(lambda: [])
         source_id = 'flow_{}'.format(uuid.uuid4().hex)
+        metadata = defaultdict(lambda: "")
+        metadata['network'] = network_name_translate(self.env.network.name.split('_20')[0])
```

Comment:
the random number after the actual network name always starts with the date, so it always start with '_20' as in 20xx year. This is basically a hack to avoid using regex. It's for the next 80 years though haha.

Reply:
ok cool, so as long as we don't have '_20' in the network name, we're good